DigitalLearningSolutions.Data.Tests/Services/CustomPromptsServiceTests.cs

```diff
@@ -0,0 +1,44 @@
+﻿namespace DigitalLearningSolutions.Data.Tests.Services
+{
+    using DigitalLearningSolutions.Data.Services;
+    using DigitalLearningSolutions.Data.Tests.Helpers;
+    using FakeItEasy;
+    using FluentAssertions;
+    using FluentAssertions.Execution;
+    using NUnit.Framework;
+
+    public class CustomPromptsServiceTests
+    {
+        private ICustomPromptsService customPromptsService;
+        private ICustomPromptsDataService customPromptsDataService;
+
+        [SetUp]
+        public void Setup()
+        {
+            customPromptsDataService = A.Fake<ICustomPromptsDataService>();
+            customPromptsService = new CustomPromptsService(customPromptsDataService);
+        }
+
+        [Test]
+        public void GetCustomPromptsForCentreByCentreId_Returns_Populated_CentreCustomPrompts()
+        {
+            // Given
+            var options = "options";
+            var expectedPrompt1 = CustomPromptsTestHelper.GetDefaultCustomPrompt(options: options);
+            var expectedCustomerPrompts = CustomPromptsTestHelper.GetDefaultCentreCustomPrompts(customPrompt1: expectedPrompt1);
+            A.CallTo(() => customPromptsDataService.GetCentreCustomPromptsByCentreId(A<int>._))
```

Comment:
Could we specify the argument on the mock as the centre id below, just to make the mock more accurate? :) 

`A.CallTo(() => customPromptsDataService.GetCentreCustomPromptsByCentreId(2))...`